backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/adempiere/ad/dao/impl/TypedSqlQuery.java

```diff
@@ -514,6 +514,8 @@ else if (postQueryFilter.accept(model))
 		finally
 		{
 			DB.close(rs, pstmt);
+			rs = null;
+			pstmt = null;
```

Comment:
futile, but let it there now.

Reply:
It's the first thing I tried  when I saw all the other methods had this. But I saw it's irrelevant, Taking it out